### ISSUE TITLE
to correct the error in "Adding validation rules to the movie model" ...

### ISSUE THREAD

**Issue by bill1109** [aspnet/Docs#7739]
First line of movies.cs sample code has been commented out... remove the // and it will work just fine.... someone must have maliciously altered the tutorial!!!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 39b6d0f3-9bc7-d091-0d45-9fc1e2e7aac8
* Version Independent ID: 20e7eb53-215b-09ab-205d-d20177221278
* Content: [Add validation to an ASP.NET Core Razor Page](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/validation?view=aspnetcore-2.1)
* Content Source: [aspnetcore/tutorials/razor-pages/validation.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/razor-pages/validation.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
No, it was added when the file was created on #6562 ...

https://github.com/aspnet/Docs/pull/6562/files#diff-4a1c6cb8d18abd380d46959b07d80cceR1

I think there's just an instruction missing that tells the reader to activate the line. It's presumably there to keep that code in the sample (validation) from become active until the reader reaches this point in the tutorial.

**Comment by bill1109** [comment]
if so, the “missing instruction” should be added !!!

**Comment by guardrex** [comment]
Yes, I agree. @Rick-Anderson authored this one, so it's best if he take care of it.

**Comment by Rick-Anderson** [comment]
snippet incorrect
![image](https://user-images.githubusercontent.com/3605364/43052005-424b1296-8dde-11e8-9881-7ac73e893d70.png)
